src/sass/elements/responsive-embed.sass

```diff
@@ -0,0 +1,26 @@
+// Inspired by bootstrap 4.0
+.embed-responsive
+  position: relative
+  display: block
+  width: 100%
+  padding: 0
+  overflow: hidden
+  // 16 by 9 ratio
```

Comment:
Declaring the padding-top second time again for easy updates in the future. iPhone X ratio might be the next from the current 16:9